src/styles.scss

```
@@ -549,18 +555,19 @@ body {
     }
 
     .mat-column-info {
-      max-width: 200px;
+      @include m.table-column-fixed(13rem);
       align-self: start;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)